main-command/src/main/scala/sbt/Command.scala

```diff
@@ -187,6 +187,13 @@ object Command {
 
   def spacedC(name: String, c: Parser[Char]): Parser[String] =
     ((c & opOrIDSpaced(name)) ~ c.+) map { case (f, rem) => (f +: rem).mkString }
+
+  implicit class CommandWithName(val cmd: Command) extends AnyVal {
```

Comment:
Coming back to this.. why the hell did I suggest an extension method (in https://github.com/sbt/sbt/pull/3531#discussion_r139164742) when the target type (`Command`) is right here? Oops.

Sorry @miklos-martin. Could you move this to `Command` please?

Reply:
Sure.
Should I move it in the `Command` trait exactly as it is here, or with a `None` implementation in the trait and override it in `SimpleCommand` with `Some(name)`?

Reply:
ping @dwijnand

Reply:
Sorry, as it is is better (imo).